File: AppGenerateFiles/dbo/Model/Catalogo_Departamento.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Catalogo_Departamento : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? id_departamento { get; set; }
       public string? nombre { get; set; }
       public int? id_nacionalidad { get; set; }
       public int? ponderacion { get; set; }
       public int? puntaje { get; set; }
       public string? clasificacion { get; set; }
       [ManyToOne(TableName = "Catalogo_Nacionalidad", KeyColumn = "id_nacionalidad", ForeignKeyColumn = "id_nacionalidad")]
       public Catalogo_Nacionalidad? Catalogo_Nacionalidad { get; set; }
       [OneToMany(TableName = "Catalogo_Clientes", KeyColumn = "id_departamento", ForeignKeyColumn = "id_departamento")]
       public List<Catalogo_Clientes>? Catalogo_Clientes { get; set; }
       [OneToMany(TableName = "Catalogo_Municipio", KeyColumn = "id_departamento", ForeignKeyColumn = "id_departamento")]
       public List<Catalogo_Municipio>? Catalogo_Municipio { get; set; }
       [OneToMany(TableName = "Condicion_Laboral_Cliente", KeyColumn = "id_departamento", ForeignKeyColumn = "id_departamento")]
       public List<Condicion_Laboral_Cliente>? Condicion_Laboral_Cliente { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Catalogo_Departamento.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools, BasicStates } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Catalogo_Nacionalidad }  from './Catalogo_Nacionalidad.js'
import { Catalogo_Clientes }  from './Catalogo_Clientes.js'
import { Catalogo_Municipio }  from './Catalogo_Municipio.js'
import { Condicion_Laboral_Cliente }  from './Condicion_Laboral_Cliente.js'
class Catalogo_Departamento extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ id_departamento;
   /**@type {String}*/ nombre;
   /**@type {Number}*/ ponderacion;
   /**@type {Number}*/ puntaje;
   /**@type {String}*/ clasificacion;
   /**@type {Catalogo_Nacionalidad} ManyToOne*/ Catalogo_Nacionalidad;
   /**@type {Array<Catalogo_Clientes>} OneToMany*/ Catalogo_Clientes;
   /**@type {Array<Catalogo_Municipio>} OneToMany*/ Catalogo_Municipio;
   /**@type {Array<Condicion_Laboral_Cliente>} OneToMany*/ Condicion_Laboral_Cliente;
}
export { Catalogo_Departamento }
